DuckDuckGo.Fluent.Plugin/QrFunctions.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Avalonia.Controls;
+using Avalonia.Media.Imaging;
+using Avalonia.Threading;
+using Blast.API.Core.Processes;
+using Blast.API.Graphics;
+using Blast.API.Processes;
+using Blast.Core.Results;
+using SoftCircuits.HtmlMonkey;
+using static DuckDuckGo.Fluent.Plugin.JsonResult;
+using static DuckDuckGo.Fluent.Plugin.AppFunctions;
+using static DuckDuckGo.Fluent.Plugin.DuckDuckGoSearchResult;
+
+namespace DuckDuckGo.Fluent.Plugin
+{
+    public class QrFunctions
+    {
+        public static async Task<DuckDuckGoSearchResult> GetQrImage(string searchedText)
+        {
+            string url = GetEndpointUrl(searchedText, true);
+            DuckDuckGoApiResult apiResult = await HttpCalls.GetApiResult(url);
+
+            if (apiResult == null) return null;
+
+            const string resultType = "QR";
+
+            if (!apiResult.AnswerType.Equals("qrcode") || string.IsNullOrWhiteSpace(apiResult.Answer)) return null;
+
+            string info = apiResult.Answer;
+
+            BitmapImageResult bitmapImageResult;
+            HtmlDocument document = HtmlDocument.FromHtml(info);
+            IEnumerable<HtmlElementNode> nodes = document.Find("img");
+            HtmlElementNode htmlElementNode = nodes.ElementAt(0);
+
+            if (htmlElementNode.Attributes.Count == 0)
```

Comment:
This `if` is not necessary, you can just initialize `bitmapImageResult` with `null` and reverse the `if` to `if (htmlElementNode.Attributes.Count > 0)`